test/test_ops.py

```diff
@@ -7,12 +7,54 @@
 import numpy as np
 import pytest
 import torch
+import torch.fx
 from common_utils import needs_cuda, cpu_and_gpu, assert_equal
 from PIL import Image
 from torch import nn, Tensor
 from torch.autograd import gradcheck
 from torch.nn.modules.utils import _pair
 from torchvision import models, ops
+from torchvision.models.feature_extraction import get_graph_node_names
+
+
+class RoIOpTesterModuleWrapper(nn.Module):
```

Comment:
@pmeier could I get your input here please. the reason I define all these  `*Wrapper` classes is that we need wrappers with variable numbers of `forward` arguments. Initially we considered using `*args` in the `forward` method but this is not traceable. Could you think of any other option for this case, or it looks ok as it it?

Reply:
@pmeier The bat sign is out. We need your wisdom :)

Reply:
I've talked with @jdsgomes offline and I don't think there is way to make this pythonic *and* appease torchscript. We probably need to go with the current design.

Reply:
After discussing offline with @pmeier we agreed that the best action is to merge the PR as there is not obvious scriptable alternative to this one.